main/conceptual/js-programming/notifiers.md

```diff
@@ -153,39 +153,39 @@ Zoe's updates for an offer show the current allocation that will be paid if the
 
 Individual contracts can use notifiers and subscriptions to provide updates giving current prices or other contract-specific details.
 
-The following methods use or return notifiers. Click on the name to go to their
+The following methods use or return notifiers. Click on the method name to go to their
```

Comment:
can we get rid of "Click on ..." while were at it?

[Avoid talking about mechanisms](https://www.w3.org/Provider/Style/NoMechanics.html)

Reply:
Good catch. The existing docs talk a lot about mechanisms, and I tried to get rid of those. I missed this one. (That sentence is now removed.)